packages/nimble-components/src/table/specs/README.md

```diff
@@ -98,23 +98,16 @@ _Feature APIs_
 The various APIs/features of the `nimble-table` will be split up amongst several different HLD documents. This section will serve to list them out and link to them as they become available:
```

Comment:
Will this spec serve as the only documentation for the column API or do we anticipate having a dedicated documentation page? If the latter, is that work tracked somewhere?

Reply:
I will create a separate issue for that, though its possible that this document could serve that purpose.